.travis.yml

```diff
@@ -7,6 +7,7 @@ services:
 python:
 - '2.7'
 cache:
+  yarn: true
```

Comment:
This is [supported by Travis](https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn). I'll update the other projects to use it.